TrackingTools/DetLayers/interface/MeasurementEstimator.h

```diff
@@ -75,12 +77,18 @@ class MeasurementEstimator {
 
   float maxSagitta() const { return m_maxSagitta;}
   float	minTolerance2() const { return m_minTolerance2;}
-
+  float	minPt2ForHitRecoveryInGluedDet() const { return m_minPt2ForHitRecoveryInGluedDet;}
 
 private:
+  /*
+   *  why here? 
+   * MeasurementEstimator is the only configurable item that percolates down to geometry event by event (actually hit by hit) and not at initialization time
+   * It is therefore the natural candidate to collect all parameters that affect pattern-recongnition 
+   * and require to be controlled with higher granularity than job level (such as iteration by iteration)
+   */ 
   float m_maxSagitta=-1.; // maximal sagitta for linear approximation
   float m_minTolerance2=100.; // square of minimum tolerance ot be considered inside a detector
-
+  float m_minPt2ForHitRecoveryInGluedDet=std::numeric_limits<float>::max();
```

Comment:
OK, the default is infty here, compared to 81X where it's set to HIP-enabled.

The downside is that the conversions (and maybe some more code) can not get their HIP mitigation applied (due to explicit construction of the estimator without needed parameters)